### PR TITLE
Don't show infobox for incomplete vaccination if Booster vaccination certificate is available (EXPOSUREAPP-9176)

### PR DESCRIPTION
**Pull request by marcauberer**
Addresses user story [EXPOSUREAPP-9176](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9176).

How to test:
- `cwa vc gen -s v.0.dn=1 v.0.sd=2 -e wru --seed mm`
- `cwa vc gen -s v.0.dn=2 v.0.sd=2 -e wru --seed mm`
- `cwa vc gen -s v.0.dn=3 v.0.sd=2 -e wru --seed mm` (Booster)

*Note: Untested, but it should work*

### CODE REVIEW

**Review by BMItr — DISMISSED** [review]
pic

lgtm

**Comment by mtwalli** [comment]
> pic
> 
> lgtm

I think the note should disappear , right ?

**Comment by BMItr** [comment]
> > pic
> > lgtm
> 
> I think the note should disappear , right ?

the infobox with incomplete status is gone. The message is about "everything is ok in some days"..

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt [hunk 1]
```diff
@@ -68,7 +68,7 @@ data class VaccinatedPerson(
     }
 
     private fun getNewestFullDose(): VaccinationCertificate? = vaccinationCertificates
-        .filter { it.doseNumber == it.totalSeriesOfDoses }
+        .filter { it.doseNumber >= it.totalSeriesOfDoses }
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt:71]: I just aligned with Martin Hermes ,whenever we have a booster vaccination , the vaccination should be considered `complete` and `immune`. in this it covering only `complete`

Comment (marcauberer) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinatedPerson.kt:73]: I adjusted it, so a person is considered as immune when he/she gets a booster, so the info box should be gone now. Is this fine?
Please test it. I am currently unable to test myself, unfortunately.

**Review by mtwalli — APPROVED** [review]
Lgtm , thanks

**Review by BMItr — APPROVED** [review]

**Pull request merged**